docs/api/accordion.md

```diff
@@ -103,11 +131,17 @@ If you would like to manage the icon yourself or use an icon that is not an `ion
 
 Regardless of which option you choose, the icon will automatically be rotated when you expand or collapse the accordion.
 
-TODO Playground
+import Icons from '@site/static/usage/accordion/customization/icons/index.md';
+
+<Icons />
 
 ### Theming
 
-TODO Playground
+Since `ion-accordion` acts as a shell around the header and content elements, you can easily theme the accordion however you would like. You can theme the header by targeting the slotted `ion-item`. Since you are using `ion-item`, you also have access to all of the [ion-item CSS Variables](./item#css-custom-properties) and [ion-item Shadow Parts](./item#css-shadow-parts). Theming the content is also easily achieved by targeting the element that is in the `content` slot.
+
+import Theming from '@site/static/usage/accordion/customization/theming/index.md';
+
+<Theming />
```

Comment:
Re: the Angular issue, you can always paste all the imports that were there originally? The popover styling playground does that, for example.

Reply:
thanks, I'll do that for now